eng/scripts/Create-Apiview-Token-Files.ps1

```diff
@@ -27,7 +28,7 @@ if ($reviews -ne $null)
 
         $pkgWorkingDir = Join-Path -Path $WorkingDir $r.ReviewID | Join-Path -ChildPath $r.RevisionID
         $codeDir = New-Item -Path $pkgWorkingDir -ItemType Directory
-        $sourcePath = $StorageBaseUrl + "/" + $r.FileID + $ContainerSas
+        $sourcePath = $StorageBaseUrl + "/" + $r.FileID + "?"+ $ContainerSas
```

Comment:
Does the secret include the `?`? I know some of them already include it in the KV secret itself.

Reply:
Nope. I have checked it. This one doesn't include it.